packages/position/src/react/__stories__/index.story.js

```diff
@@ -1,139 +1,220 @@
 import React from 'react'
-import { storiesOf } from '@storybook/react'
-import Tooltip from '@pluralsight/ps-design-system-tooltip/react'
+
+import core from '@pluralsight/ps-design-system-core'
+import Tooltip from '@pluralsight/ps-design-system-tooltip/react.js'
 
 import * as positionFns from '../../js/index.js'
 import * as positionComponents from '../index.js'
 
-const Box = React.forwardRef((props, ref) => (
-  <div
-    ref={ref}
-    style={{
-      height: '200px',
-      width: '200px',
-      border: `4px dashed ${core.colors.pink}`,
-      display: 'flex',
-      alignItems: 'center',
-      justifyContent: 'center',
-      textAlign: 'center',
-      color: core.colors.pink,
-      fontSize: core.type.fontSizeMedium,
-      fontWeight: core.type.fontWeightBold,
-      ...props.style
-    }}
-  >
-    {props.children}
-  </div>
-))
-Box.propTypes = {
-  children: PropTypes.node,
-  style: PropTypes.object
+const Box = React.forwardRef((props, forwardedRef) => {
+  const ref = React.useRef()
+  React.useImperativeHandle(forwardedRef, () => ref.current)
+
+  const className = css({
+    alignItems: 'center',
+    border: `4px dashed ${core.colors.pink}`,
+    color: core.colors.pink,
+    display: 'flex',
+    fontSize: core.type.fontSizeMedium,
+    fontWeight: core.type.fontWeightBold,
+    height: '200px',
+    justifyContent: 'center',
+    textAlign: 'center',
+    width: '200px'
+  })
+
+  return <div className={className} ref={ref} {...props} />
+})
+
+const MockToolip = React.forwardRef((props, forwardedRef) => {
+  const ref = React.useRef()
+  React.useImperativeHandle(forwardedRef, () => ref.current)
+
+  return (
+    <Tooltip ref={ref} {...props}>
+      The tip
+    </Tooltip>
+  )
+})
+
+const ScrollContainer = props => {
+  const containerClass = css({
+    border: `4px dashed ${core.colors.orange}`,
+    color: core.colors.bone,
+    height: 500,
+    overflow: 'scroll',
+    padding: 20,
+    width: 500
+  })
+  const shimClass = css({
+    border: `1px dashed ${core.colors.bone}`,
+    height: 200,
+    margin: '20px 0',
+    opacity: 0.4
+  })
+
+  return (
+    <div className={containerClass} {...props}>
+      <div className={shimClass} />
+      {props.children}
+      <div className={shimClass} />
+    </div>
+  )
 }
+ScrollContainer.propTypes = { children: PropTypes.node }
 
-// NOTE: old style refs to make storybook/storyshots happy
-class JsStory extends React.Component {
-  constructor(props) {
-    super(props)
+const basicStories = storiesOf('Components | Position / basic', module)
+Object.values(positionComponents).forEach(Comp => {
+  const { displayName } = Comp
+  const name = `<${Comp.displayName} />`
 
-    this.state = { style: {} }
-    this.tooltipRef = React.createRef()
+  basicStories.add(displayName, () => (
+    <Comp show={<MockToolip />}>
+      <Box>{name}</Box>
+    </Comp>
+  ))
+})
+
+const portalStories = storiesOf('Components | Position / portals', module)
```

Comment:
The portal story is still looking messed up to me (not centered where it should be).  Does it look this way for you?

<img width="442" alt="Screen Shot 2019-10-01 at 11 21 21 AM" src="https://user-images.githubusercontent.com/552044/65985446-12bd9e00-e43f-11e9-9ac2-f79dc6d2f866.png">

Reply:
The site portal example is also messed up:

<img width="401" alt="Screen Shot 2019-10-01 at 11 20 30 AM" src="https://user-images.githubusercontent.com/552044/65985498-2cf77c00-e43f-11e9-83d1-cec70a9a2a50.png">

Reply:
weird. i'm not seeing the issue. i guess can you bootstrap and double check. i've noticed on the docs site it can take a sec to get the correct position - i assumed that was a nextjs in dev mode issue 🤷‍♂

Reply:
POSITION ISSUE IN STORYBOOK:
Just updated the branch, clean, bootstrap, checked storybook for Position in ff, chrome, safari, brave.  They'll still look like the storybook screenshot above.  Does yours? 

(To clarify, it sounds like you were saying the *site* didn't look like the screenshot I posted above.  How about your storybook?  Does it match the problem screenshot?)

POSITION ISSUE IN SITE:
I also double-checked the site portal render in these browsers.  They all show the same as the problem screenshot above, except for Safari, which is a bit diff:

<img width="292" alt="Screen Shot 2019-10-04 at 1 55 44 PM" src="https://user-images.githubusercontent.com/552044/66236210-148d9880-e6af-11e9-8a3a-8c7a35538ab5.png">

And then I noticed the the "js example" below it is also messed up (supposed to be `rightOf`):

<img width="281" alt="Screen Shot 2019-10-04 at 1 56 41 PM" src="https://user-images.githubusercontent.com/552044/66236220-2111f100-e6af-11e9-8549-5989c03bfa12.png">

ISSUE WITH SSR TYPEAHEAD:
Finally, I thought I'd build the site so we could test the hypothesis about nextjs dev server being part of the problem.  When I built, I got:

```
Error occurred prerendering /components/typeahead: ReferenceError: document is not defined
    at usePortal (/Users/jaketrent/dev/design-system/packages/typeahead/dist/react/index.js:277:35)
```

So I think there probably needs to be some conditional around the use of `document` in Typehaead for SSR support.